Lib/platform.py

```diff
@@ -746,7 +749,10 @@ def from_subprocess():
         """
         Fall back to `uname -p`
         """
-        import subprocess
+        try:
+            import subprocess
+        except ImportError:
+            return
```

Comment:
Might as well be explicit.

```suggestion
            return None
```